api/v1beta1/workspace_validation.go

```diff
@@ -440,13 +447,13 @@ func (i *InferenceSpec) validateCreate(ctx context.Context, namespace string) (e
 	}
 
 	if i.Config != "" {
```

Comment:
If config is not filled explicitly, we will use `inference-params-template` this file if exist in the same ns by default.
Can you add this logic as well?

Reply:
following up offline on more requirements needed for this